assets/js/modules/search-console/components/dashboard/SearchConsoleDashboardWidgetKeywordTable.js

```diff
@@ -24,68 +24,66 @@ import { map } from 'lodash';
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
-import { getTimeInSeconds, numberFormat, getModulesData } from '../../../../util';
+import Data from 'googlesitekit-data';
+import { getTimeInSeconds, numberFormat } from '../../../../util';
 import withData from '../../../../components/higherorder/withdata';
 import { TYPE_MODULES } from '../../../../components/data';
 import { getDataTableFromData, TableOverflowContainer } from '../../../../components/data-table';
 import PreviewTable from '../../../../components/preview-table';
+import { STORE_NAME } from '../../datastore/constants';
+const { useSelect } = Data;
 
-class SearchConsoleDashboardWidgetKeywordTable extends Component {
-	render() {
-		const { data } = this.props;
+const SearchConsoleDashboardWidgetKeywordTable = ( props ) => {
+	const { data } = props;
+	const domain = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+	const baseServiceURL = useSelect( ( select ) => select( STORE_NAME ).getServiceURL( { path: '/performance/search-analytics' } ) );
+	const headers = [
+		{
+			title: __( 'Keyword', 'google-site-kit' ),
+			tooltip: __( 'Most searched for keywords related to your content', 'google-site-kit' ),
+			primary: true,
+		},
+		{
+			title: __( 'Clicks', 'google-site-kit' ),
+			tooltip: __( 'Number of times users clicked on your content in search results', 'google-site-kit' ),
+		},
+		{
+			title: __( 'Impressions', 'google-site-kit' ),
+			tooltip: __( 'Counted each time your content appears in search results', 'google-site-kit' ),
+		},
+	];
+	const links = [];
 
-		const headers = [
-			{
-				title: __( 'Keyword', 'google-site-kit' ),
-				tooltip: __( 'Most searched for keywords related to your content', 'google-site-kit' ),
-				primary: true,
-			},
-			{
-				title: __( 'Clicks', 'google-site-kit' ),
-				tooltip: __( 'Number of times users clicked on your content in search results', 'google-site-kit' ),
-			},
-			{
-				title: __( 'Impressions', 'google-site-kit' ),
-				tooltip: __( 'Counted each time your content appears in search results', 'google-site-kit' ),
-			},
+	const dataMapped = map( data, ( row, i ) => {
+		const query = row.keys[ 0 ];
+		links[ i ] = addQueryArgs( baseServiceURL, { query, resource_id: domain, num_of_days: 28 } );
```

Comment:
See above, the `resource_id: domain` could also be part of `baseServiceURL`.

Reply:
Updated. I also moved `num_of_days` as well for the same reason.